src/index.js

```diff
@@ -13,9 +13,9 @@ export default options => {
 
   /**
    * Get rooms socket is subscribed to.
-   *
-   * @param {String} id
-   * @param {Function} [fn]
+   * Used for get rooms
+   * @param {String} id - spark's ID
+   * @param {Function} [fn] - callback
```

Comment:
I had the same issue regarding the promises support for older node versions, so didnt do it. But since we are dropping the older version will update it to use promises :)